src/layouts/layout.jsx

```diff
@@ -57,9 +59,50 @@ export const StyledFooterText = styled.label`
   line-height: 38px;    
 `
 
+const filters = [
+  {name: "all", displayName: "All"},
+  {name: "development", displayName: "Development"},
+  {name: "product-design", displayName: "Design"},
+  {name: "machine-learning", displayName: "Machine Learning"},
+  {name: "blockchain", displayName: "Blockchain"},
+  {name: "people-events", displayName: "People"},
+];
+
+const getPath = () => {
+  if (typeof window !== 'undefined') {
+    if (window.location.pathname === '/') {
+      return 'all';
+    }
+    return filters.find(({ name }) => window.location.href.includes(name))?.name;
+  }
+  return 'all';
+}
+
 function Layout({ children }) {  
   const [contextState, setContextState] = useState(initialState);
   const isMobile = useMediaQuery("(max-width: 768px)");
+  const [ category, setCategory ] = useState('all');
+
+  useEffect(() => {
+    const windowScrollTimeout = setTimeout(() => {
+      window.scrollTo(0, 0)
+      clearTimeout(windowScrollTimeout)
```

Comment:
clearTimeout is not needed here. because it already gots timeout at this point, there's nothing to clear